pkg/operator/webhooksupportabilitycontroller/degraded_webhook_admission.go

```diff
@@ -27,6 +27,7 @@ func (c *webhookSupportabilityController) updateMutatingAdmissionWebhookConfigur
 				Name:                  webhook.Name,
 				CABundle:              webhook.ClientConfig.CABundle,
 				FailurePolicyIsIgnore: webhook.FailurePolicy != nil && *webhook.FailurePolicy == admissionregistrationv1.Ignore,
+				TimeoutSeconds:        webhook.TimeoutSeconds,
```

Comment:
Does it make sense to perform defaulting here rather than as part of every check?

Reply:
The defaulting is cheap. In the future we might consider adding some logs.
In addition to that it is consistent with defaulting the `port`.

Reply:
Right, I'm not worried about the cost, I think it would be a tidier separation of responsibility. Since we already translate the API object into an internal representation for use by the dial probes, it didn't make sense to me that the internal representation (i.e. `webhookInfo`) wasn't directly usable by the dial probe.

It's only my preference and I'm satisfied with consistency too.